openedx/core/djangoapps/global_statistics/tasks.py

```diff
@@ -35,11 +37,16 @@ def count_data():
     
     "Get courses amount within current platform."
     courses_amount = len(modulestore().get_courses())
+
+    "Secret token to authorize our platform on remote server"
+    secret_token, created = TokenStorage.objects.get_or_create(pk=1)
     
     "Posting data to receiving server."
     data_to_send = requests.post(PERIODIC_TASK_POST_URL, data={
     	'courses_amount': courses_amount,
     	'students_amount': students_amount,
     	'latitude': latitude,
-    	'longitude': longitude
+    	'longitude': longitude,
+    	'platform_url': SITE_NAME,
+    	'secret_token': secret_token
```

Comment:
Please send token in `Authorization` header
see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization